mep/books/models.py

```diff
@@ -262,10 +262,15 @@ def authors(self):
         '''work creators with type author'''
         return self.creator_by_type('Author')
 
+    @property
     def author_list(self):
         '''semicolon separated list of author names'''
         return '; '.join([auth.name for auth in self.authors])
-    author_list.verbose_name = 'Authors'
```

Comment:
The verbose name and associated fact that this was a method and not a property is because we're using this in the admin list view for works. Seems like the shorter label would still be nice to have.

Reply:
gotcha, I'll switch it back